client/js/views/Lobby.vue

```diff
@@ -5,7 +5,12 @@
         <img id="logo" alt="Cuttle logo" src="../img/logo.png" />
       </v-col>
       <v-col md="8" class="my-auto">
-        <h1>Lobby for {{ gameName }}</h1>
+        <h1>
+          Lobby for {{ gameName }}
+          <small v-if="isRanked" class="lobby-ranked-text">
+            (Ranked <v-icon v-if="isRanked" class="ml-1" medium>mdi-trophy</v-icon>)
```

Comment:
Not gonna die on this hill but I pref multiples of 8px (1 unit in vuetify's spacing classes = 4px)
```suggestion
            (Ranked <v-icon v-if="isRanked" class="ml-2" medium>mdi-trophy</v-icon>)
```

Reply:
I thought it looked uglier with 8px, but I don't care enough to argue, hahaha.

Reply:
I tried it again and I refuse, it looks like absolute shit with the larger space.
![image](https://user-images.githubusercontent.com/414475/194535620-970d5aea-01b2-4319-98a2-36a3fe20fcf2.png)

I ended up just taking it away altogether, haha.
![image](https://user-images.githubusercontent.com/414475/194535719-056cfdd4-0b82-40d0-9f3a-7ea9fb4b4fce.png)

Reply:
It might be worth asking Sarankit to take a pass on the lobby screen and just redesign it. The groundwork here is good but the design is limping along with the bare minimum :)